parser/linpeas-parser.py

```diff
@@ -0,0 +1,240 @@
+SOFTWARE_INFORMATION_SLUG = "software-information"
+SYSTEM_INFORMATION_SLUG = "system-information"
+AVAILABLE_SOFTWARE_SLUG = "available-software"
+NETWORK_INFORMATION_SLUG = "network-information"
+USERS_INFORMATION_SLUG = "users-information"
+INTERESTING_FILES_SLUG = "interesting-files"
+
+try:
+    linpeas_output_path = sys.argv[1]
+except IndexError as err:
+    # You can pipe the output to "jq" if you have setup 
+    print("Error: Needs to pass the .out file\n./linpeas-parser.py <output_file>")
+    sys.exit()
+
+def basic_information_parser(json_output: dict, row: str) -> dict:
+    """Returns a dict
+
+    Parses a row a following the boundaries of basic information
+    """
+    parsed_row = {}
+
+    if ":" in row:
+        parsed_row = {"label": row.replace(":", " ").strip()}
+    elif "[+]" in row:
+        parsed_row = {"label": row.replace("[+]", "").strip()}
+    else: 
+        parsed_row = {"label": row}
+
+    return parsed_row
+ 
+def software_information_parser(json_output: dict, row: str) -> dict:
+    """Returns a dict
+
+    Parses a row a following the boundaries of software information
+    """
+    return {"row": row}
+
+def system_information_parser(json_output: dict, row: str) -> dict:
+    """Returns a dict
+
+    Parses a row a following the boundaries of system information
+    """
+    
+    return {"row": row}
+
+def available_software_parser(json_output: dict, row: str) -> dict:
+    """Returns a dict
+
+    Parses a row a following the boundaries of available software
+    """
+    return {"row": row}
+
+def network_information_parser(json_output: dict, row: str) -> dict:
+    """Returns a dict
+
+    Parses a row a following the boundaries of network information
+    """
+    return {"row": row}
+
+def users_information_parser(json_output: dict, row: str) -> dict:
+    """Returns a dict
+
+    Parses a row a following the boundaries of network information
+    """
+    return {"row": row}
+
+def interesting_files_parser(json_output: dict, row: str) -> dict:
+    """Returns a dict
+
+    Parses a row a following the boundaries of network information
+    """
+    return {"row": row}
+
+def get_parser_by_slug(slug: str):
+    """Returns a function
+
+    Returns the right parser based on the slug
+    """
+    return parsers[slug]
+
+parsers = {
+    BASIC_INFORMATION_SLUG: basic_information_parser,
+    SOFTWARE_INFORMATION_SLUG: software_information_parser,
+    SYSTEM_INFORMATION_SLUG: system_information_parser,
+    AVAILABLE_SOFTWARE_SLUG: available_software_parser,
+    NETWORK_INFORMATION_SLUG: network_information_parser,
+    USERS_INFORMATION_SLUG: users_information_parser,
+    INTERESTING_FILES_SLUG: interesting_files_parser,
+}
+
+def read_file(output_path: str) -> [str]:
+    """Returns a list of strings
+
+    Reads file from a specich path and returns it as a list
+    """
+    with open(output_path, 'r') as file:
+        return [row.strip() for row in file.readlines() if row]
```

Comment:
No need for a `with` statement since you'll `return` anyway (and the file should auto-close anyway), so you can do:
```python3
    return [row.strip() for row in open(output_path, 'r').readlines() if row]
```